File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.*;
import java.util.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        int T = Integer.parseInt(br.readLine());
        
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        
        StringTokenizer st = new StringTokenizer(br.readLine());
        for(int i = 0; i < T; i++) {
            int num = Integer.parseInt(st.nextToken());
            
            if(min > num) min = num;
            if(max < num) max = num;
        }
        
        System.out.println(min + " " + max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.StringTokenizer

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val n = Integer.parseInt(br.readLine())
    val arr = mutableListOf<Int>()
    val st = StringTokenizer(br.readLine())
    for (i in 0 until n){
        arr.add(Integer.parseInt(st.nextToken()))
    }
    val max = arr.max()
    val min = arr.min()
    print("$min $max")
}